content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -276,6 +276,33 @@ not affect the performance of the member.
 
 Below is an example for taking a snapshot of the keyspace served by
 `$ENDPOINT` to the file `snapshot.db`:
+step1:Take an etcd snapshot backup using the following command
+
+ETCDCTL_API=3 etcdctl \
+  --endpoints=https://127.0.0.1:2379 \
+  --cacert=<ca-file> \
+  --cert=<cert-file> \
+  --key=<key-file> \
+  snapshot save <backup-file-location>
```

Comment:
This should be a block of code - see https://www.google.com/search?q=fenced+code+block+in+Markdown